### DISCUSSION TITLE
Can keyboard shortcuts be changed

### DISCUSSION THREAD

**Discussion by ceverson70** [TabbyML/tabby#1572] [Ideas]
On Mac the keyboard shortcut control + right arrow is already in use to switch full screen tabs/spaces. But i don't see an option to change the one tabby uses, or a feature request to allow users to select. Maybe i missed it or maybe I am missing something

**Comment by wsxiaoys** [comment]
it's actually a bug., tracked at https://github.com/TabbyML/tabby/issues/1557

cc @icycodes
